qa/standalone/ceph-helpers.sh

```diff
@@ -2055,6 +2061,15 @@ function inject_eio() {
     done
 }
 
+function multidiff() {
+    if ! diff $@ ; then
```

Comment:
nit, i wonder if we should redirect the output of `diff $@`  to `/dev/null` here? otherwise we will have double dose of `diff` in the output.

Reply:
The reason we need both is that the side by side diff won't wrap lines, so it can cut off output.  So the side by side one tells you where the problem is and if it is cut off the regular diff shows all of the line(s).